index.html

```diff
@@ -279,8 +279,18 @@ <h1>
     </p>
 
     <p>
-This design eliminates dependence on centralized registries for identifiers as
-well as centralized certificate authorities for key management, which is the
+This specification utilizes registries to enable interoperable definitions of
+identifiers used by DID implementations, including
+<a>DID document</a> member names defined by this and other specifications,
+<a>DID methods</a>,
+and cryptographic algorithm identifiers.
+Some of these registries are operated by the W3C DID Working group
+and others are operated by other standards organizations, including the IETF.
+    </p>
+
+    <p>
+This design eliminates dependence on centralized registries
+for key management, which is the
```

Comment:
Propose to keep the term "identifier" here in order to better preserve the original content of this paragraph.

```suggestion
for identifier and key management, which is the
```